recipes/r-rsaga/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "1.3.0" %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rsaga
+  version: {{ version.replace("-", "_") }}
+
+source:
+  fn: RSAGA_{{ version }}.tar.gz
```

Comment:
fn can be removed?

Reply:
Last time I tried this only work for python recipes. Let's try it.

Reply:
I'm clearly outdated and did not payed attention on the auto-generated shapefile recipe here :smile: